spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/mkldnn/AvgPoolingSpec.scala

```diff
@@ -236,4 +236,33 @@ class AvgPoolingSpec extends BigDLSpecHelper {
 
     Equivalent.nearequals(seq.output.toTensor, seq2.output.toTensor, 1e-2) should be (true)
   }
+
+  "global average pooling" should "work correctly" in {
```

Comment:
Add unit tests to check:
1. whether dnn global average pooling equals blas.
2. whether converting global pooling blas to dnn can work correctly

Reply:
OK.